code/modules/clothing/suits/armor.dm

```diff
@@ -442,6 +442,16 @@
 		disable(rand(2, 5))
 		return TRUE
 
+/obj/item/clothing/suit/armor/reactive/random //Spawner for random reactive armor
+	name = "Random Reactive Armor"
+
+/obj/item/clothing/suit/armor/reactive/random/New()
+	..()
+	var/list/types = subtypesof(/obj/item/clothing/suit/armor/reactive)
```

Comment:
```suggestion
	var/list/types = subtypesof(/obj/item/clothing/suit/armor/reactive) - /obj/item/clothing/suit/armor/reactive/random
```
one other nitpick, we need exclude the `/random` subtype from this list. It won't cause issues in the game b/c eventually the non-random subtype will roll but yeah.